examples/php/cli.php

```diff
@@ -4,13 +4,20 @@
 date_default_timezone_set('UTC');
 
 
-$root = dirname(dirname(dirname(__FILE__)));
-include $root . '/ccxt.php';;
+$ccxtroot = dirname(dirname(dirname(__FILE__)));	//when this is in $ccxtroot/examples/php/cli.php
+$root = dirname($ccxtroot);
+if(basename($root) == 'ccxt'){$root = dirname($root);}
+if(basename($root) == 'vendor'){$autoloadFile = $root.DIRECTORY_SEPARATOR.'autoload.php';}
+else{$autoloadFile = $root. DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php';}
+if (file_exists($autoloadFile)) {
+    require_once $autoloadFile;
+}
+include $ccxtroot . '/ccxt.php';
```

Comment:
I'm not sure why this is needed, I can use cli.php normally as is

Reply:
This is the commit in this PR that is least important or interesting.  I just thought it could be helpful to CCXT newbies for two reasons.  One is so that this example can run out of the box after a default composer installation of CCXT.  Otherwise, when autoload.php is not found, line 11/18 fails because it can't find ReactPHP. The second is as an example of how to find autoload.php from the examples directory. 

The very first line of [Install CCXT for PHP](https://docs.ccxt.com/#/Install?id=php) says to use Packagist/Composer.  Composer's default installation location for ccxt is <directory of composer.phar>/vendor/ccxt/ccxt/.  The default location of this file would be <directory of composer.phar>/vendor/ccxt/ccxt/examples/php/cli.php  So finding autoload.php requires going up a few more levels.  However, I don't want to make this a loop, as improper dirname loops can become infinite.

Reply:
@carlosmiei Please let me know if I should remove this part of the commit or entire file if the rest of the PR can be approved.  Thanks.

Reply:
yeah I would remove this for now